source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -251,13 +251,27 @@ def _read_response(response, origin, destination, mode, fallback_extremity, requ
             fm.car_no_park.name: response_pb2.Car,
         }
         section.street_network.mode = map_mode[mode]
+
+        # handle maneuvers to fill Path field
+        def _convert_direction(direction):
+            sentence = direction.lower()
+            if "right" in sentence:
+                return 90
+            elif "left" in sentence:
+                return -90
+            else:
+                return 0
+
```

Comment:
I also found an example of "direction": 113 in a street_network.

Reply:
yes for the classical Path kraken compute a angle.
This is for backward compatibility that I convert, for HERE, the `right/left` string to an angle. And I choose `+90/-90` to reflect right/left